docs/user/managing-devices.md

```diff
@@ -618,3 +618,53 @@ flightctl console <some_device_name>
 To disconnect, enter "exit" on the console. To force-disconnect, press `<ctrl>+b` three times.
 
 ## Decommissioning Devices
+
+Decommissioning a device is the proper way to unenroll it and permanently remove it from flightctl management. This is an action that cannot be undone. Decommissioning should be performed before deleting a device.
```

Comment:
Could we make a short statement on what happens / why is it important? Something like "Decommissioning a device is the proper way [...]. When the user requests decommissioning of a device, the Flight Control service signals the agent to run a decommissioning process that includes erasing its management certificate and key and with it the device's identity."

Reply:
Good idea, I've added it